src/v/kafka/server/quota_manager.cc

```diff
@@ -277,7 +277,7 @@ throttle_delay quota_manager::throttle_fetch_tp(
     auto delay_ms = throttle(
       quota_id, *target_tp_rate, now, it->second.tp_fetch_rate);
     throttle_delay res{};
-    res.first_violation = false;
```

Comment:
the removal of references to `first_violation` need to be made at or before the commit which removes that field from `throttle_delay`, otherwise the previous commits don't build.

Reply:
That's an artifact of rebasing onto conflicting changes. Fixing...